### PR TITLE
Merge initialize-project-with-components branch into development branch

### PR DESCRIPTION
**Pull request by aristides1000**
- [x] The following activities were carried out:
- [x] Initialize React app.
- [x] Add React-Redux (npm install react-redux).
- [x] Structure your application files using a "feature folder" approach and use the duck's pattern for your Redux files.
- [x] The building blocks of your app should be set as re-usable components.
- [x] Add React Router and set two <Route>s and <Link>s for the app's navigation:
- [x] Books - the default view
- [x] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet)
- [x] Should have a form for adding a book (no functionality yet)
- [x] Categories
- [x] Should display "Under construction" text only.
- [x] Styling is not required at this point.

### CODE REVIEW

File: src/pages/books.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                  e.preventDefault();
+                  createBookAction(bookCreate);
+                }
+              }
+              className="btn btn-primary"
+              type="submit"
+            >
+              Submit
+            </button>
+          </form>
```
Comment (henatan99) [inline src/pages/books.js:55]: - [ ] OPTIONAL: I think you can construct the form element in a separate component. It increases the component re-usability and app scalability. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @aristides1000 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] The react-app default readme gives general information about a react-app but you need to update it with project description, setup, installation and usage instructions, author info, and acknowledgment to the least. It is strongly advised to use the Microvers [readme template](https://github.com/microverseinc/readme-template) which you can customize for your project. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aristides1000** [comment]
Thank you very much for your comments 👍👍👍, I have already applied the changes to the README.md file.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aristides1000** [comment]
Thanks for your review Victor

**Pull request merged**